radio/src/gui/colorlcd/model_select.cpp

```diff
@@ -287,12 +287,7 @@ class ModelCategoryPageBody: public FormWindow {
                 });
               }
               menu->addLine(STR_CREATE_MODEL, [=]() {
-                  storageCheck(true);
-                  modelslist.setCurrentModel(modelslist.addModel(category, createModel()));
-#if defined(LUA)
-                  // chainMenu(menuModelWizard);
-#endif
-                  update(category->size() - 1);
+                  this->createModelAction();
```

Comment:
why this->
should be named createModel()
or all functions would be postfixed by "Action"?

Reply:
I meant to return the lambda, in fact. Thx for reminding me!